### DISCUSSION TITLE
Transaction Support

### DISCUSSION THREAD

**Discussion by GeorgeTan615** [ClickHouse/clickhouse-go#1340] [Feature requests]
Does `clickhouse-go` support real transactions now? Just like how `JDBC Driver` has [transaction support](https://clickhouse.com/docs/en/integrations/java#jdbc-driver).

**Comment by jkaflik** [comment]
Transaction is not a client feature. It is handled and supported by the client, but the way it can be achieved is described in ClickHouse documentation: https://clickhouse.com/docs/en/guides/developer/transactional
